admin-dev/themes/default/template/controllers/cart_rules/conditions.tpl

```diff
@@ -294,10 +294,10 @@
 				</a>
 			</div>
 
-		{if ($shops.unselected|@count) + ($shops.selected|@count) > 1}
+		{if ($all_shops|@count) > 1}
 			<p class="checkbox">
 				<label>
-					<input type="checkbox" id="shop_restriction" name="shop_restriction" value="1" {if $shops.unselected|@count}checked="checked"{/if} />
+					<input type="checkbox" id="shop_restriction" name="shop_restriction" value="1" {if $shops.selected|@count || ($shops.selected|@count + $shops.unselected|@count < $all_shops|@count)}checked="checked"{/if} />
```

Comment:
```suggestion
					<input type="checkbox" id="shop_restriction" name="shop_restriction" value="1" {if $shops.selected|@count || (($shops.selected|@count + $shops.unselected|@count) < $all_shops|@count)}checked="checked"{/if} />
```

Reply:
You already made a suggestion on this line https://github.com/PrestaShop/PrestaShop/pull/28153#discussion_r848056768, where I fully agreed. 
But for this suggested change, I think everybody knows how to read `a + b < c`, therefore parenthesis `(a + b) < c` are superfluous here